.circleci/workflows.yml

```diff
@@ -681,6 +685,20 @@ commands:
           steps:
             - ferrocene-setup-darwin
       - aws-oidc-auth
+      - when:
+          condition:
+            equal: [<<parameters.qnx>>, true]
+          steps:
+            - run:
+                name: Fetch QNX710 toolchain
+                command: aws s3 cp s3://ferrocene-ci-artifacts/qnx/qnx710.tar.zst - | zstd -d | tar x
```

Comment:
Can we store this in `s3://ferrocene-ci-mirrros/manual`? That's where everything else we mirrored is stored.